patroni/ctl.py

```diff
@@ -7,7 +7,7 @@
 import datetime
 import dateutil.parser
 import dateutil.tz
-import cdiff
+import ydiff
```

Comment:
```suggestion
try:
    import ydiff
except ImportError:
    import cdiff as ydiff
```

And please move it after the line with `import yaml`